tests/test_extractors.py

```diff
@@ -323,6 +324,18 @@ def test_biocam_extractor(self):
         check_recordings_equal(self.RX, RX_biocam)
         check_dumping(RX_biocam)
 
+    def test_ced_extractor(self):
+        n_elements = 881
+        trace_out = np.arange(0, n_elements, dtype=np.short)
+        assert len(trace_out) == n_elements
+        smrx_file = sp.SonFile('test.smrx')
+        smrx_file.SetWaveChannel(0, 7, sp.DataType.Adc)
+        smrx_file.WriteInts(0, trace_out, 0)
+        del smrx_file
+        recording = se.CEDRecordingExtractor('test.smrx', smrx_channel_ids=[0])
+        trace_in = recording.get_traces([0])[0]
+        assert len(trace_in) == n_elements
+
```

Comment:
Can you make this optional on having sonpy?
```suggestion
    def test_ced_extractor(self):
        try:
            from sonpy import lib as sp
            n_elements = 881
            trace_out = np.arange(0, n_elements, dtype=np.short)
            assert len(trace_out) == n_elements
            smrx_file = sp.SonFile('test.smrx')
            smrx_file.SetWaveChannel(0, 7, sp.DataType.Adc)
            smrx_file.WriteInts(0, trace_out, 0)
            del smrx_file
            recording = se.CEDRecordingExtractor('test.smrx', smrx_channel_ids=[0])
            trace_in = recording.get_traces([0])[0]
            assert len(trace_in) == n_elements
     except:
         print("sonpy not installed")
```

Reply:
sonpy is in the requirements_test.txt. So I think it is OK.
If we want to skip the test which is bad idea I think I prefer use `@pytest.skip()`